.vscode/launch.json

```diff
@@ -12,6 +12,36 @@
 				"${workspaceFolder}/taqueria-vscode-extension/out/*.js"
 			],
 			"preLaunchTask": "build-and-watch-vscode"
+		},
+		{
+			"request": "launch",
+			"name": "Launch CLI",
+			"type": "pwa-node",
+			"program": "${workspaceFolder}/index.ts",
+			"cwd": "/Users/jimmychenchen/Desktop/bugbug",
```

Comment:
Will try to parametrize them in the future. Leaving this here now won't cause any issues.